src/compiler/utilities.ts

```diff
@@ -2483,10 +2483,9 @@ namespace ts {
      * Resolves a local path to a path which is absolute to the base of the emit
      */
     export function getExternalModuleNameFromPath(host: EmitHost, fileName: string): string {
-        const getCanonicalFileName = (f: string) => host.getCanonicalFileName(f);

```

Comment:
it should be reverted back to using lambda: there are other hosts (AFAIR one of them used to be in Angular codebase) that won't work if `this` is torn off